src/web/rustdoc.rs

```diff
@@ -164,6 +161,7 @@ pub fn rustdoc_redirector_handler(req: &mut Request) -> IronResult<Response> {
         .unwrap_or_else(|_| crate_name.into())
         .into_owned();
     let req_version = router.find("version");
+    let mut target = router.find("target");
```

Comment:
I don't see a `:target` binding introduced in the routes. What is this meant to do?

Reply:
This handles this route:

https://github.com/rust-lang/docs.rs/blob/554bb0e7f40d22db3728d42e019898c4a084ba97/src/web/routes.rs#L113-L116

Which is the one hit by non-default-target search paths.

Reply:
(The lack of checking this parameter is why non-default-target searches didn't work).

Reply:
Got it, thank you!